### PR TITLE
allow a decimal format for an integer

### PR DESCRIPTION
**Pull request by mguida22**
From #199, when using a decimal format with a wildcard, on an integer, numbro gives an error

```
numbro(-40).format('#.*');
```

> numbro.js:643 Uncaught TypeError: Cannot read property 'length' of undefined

This addresses the issue by checking for a decimal before attempting to format it. The same line from above will now return `'-40'`

### CODE REVIEW

File: numbro.js [hunk 1]
```diff
@@ -658,7 +661,7 @@
 
             w = d.split('.')[0];
 
-            if (d.split('.')[1].length) {
+            if (d.split('.').length > 1 && d.split('.')[1].length) {
```
Comment (msaglietto) [inline numbro.js]: mm in here you are doing the split like 4 times .. maybe its better to save the result of the split and then use it

Comment (BenjaminVanRyseghem) [inline numbro.js]: sounds like a good idea :smile:

**Comment by msaglietto** [comment]
THanks @mguida22  I think it looks ok now ..  what do you think @BenjaminVanRyseghem  could we merge this .. I could use it =P ...

**Comment by BenjaminVanRyseghem** [comment]
thanks 👍

**Comment by mguida22** [comment]
Thanks for the feedback on this!

**Comment by BenjaminVanRyseghem** [comment]
in numbro 1.10.0

**Pull request merged**